### DISCUSSION TITLE
Transport Efficiency and the Bruggeman Correlation

### DISCUSSION THREAD

**Discussion by ch662** [pybamm-team/PyBaMM#2387] [Q&A] [answered]
Hello, can someone please explain to me how transport efficiency is explicitly defined in Pybamm? 
I saw how tortuosity was renamed to transport efficiency because it is defined differently in Pybamm, but I have had a hard time finding the exact differences. 

For example, what is the difference between transport efficiency in Pybamm and the Bruggeman correlation as defined here:
<img width="362" alt="bruggeman" src="https://user-images.githubusercontent.com/108564982/196865886-3b1c5789-a660-4569-a1b1-348085fa968d.png">

**Comment by valentinsulzer** [comment, marked as answer]
Transport efficiency is defined as porosity to the power or Bruggeman coefficient (usually 1.5)
